bitnami/postgresql/templates/NOTES.txt

```diff
@@ -55,5 +55,3 @@ To connect to your database from outside the cluster execute the following comma
 {{- include "common.warnings.rollingTag" .Values.image -}}
 
 {{- $passwordValidationErrors := include "common.validations.values.postgresql.passwords" (dict "secret" (include "postgresql.fullname" .) "context" $) -}}
-
-{{- include "common.errors.upgrade.passwords.empty" (dict "validationErrors" (list $passwordValidationErrors) "context" $) -}}
```

Comment:
We shouldn't remove this passwords validation. We can have this inside an `{{- if $secret }}` or better yet, have it call a helper that does `{{- $secret := (lookup "v1" "Secret" .Release.Namespace $secretName) }}`.